### ISSUE TITLE
space-before-keywords with shorthand properties

### ISSUE THREAD

**Issue by danielmoore** [eslint/eslint#3879]
This appears to be pretty much the same thing as #3756.

Expected success:
```js
/* eslint space-before-keywords: [2, "always"] */

var x = {
  get foo() { return 42; }.
  set foo() { }
};
```

Actual:

```
  4:10  error  Missing space before keyword "("  space-before-keywords
  5:10  error  Missing space before keyword "("  space-before-keywords
```

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by danielmoore** [comment]
Oops, looks like this was already reported in #3854
